cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -87,6 +87,23 @@ def catalog_item(appliance, provider, dialog, catalog, prov_data):
         catalog_item.delete()
 
 
+@pytest.fixture
+def catalog_bundle(appliance, dialog, catalog, catalog_item):
+    catalog_item_list = []
+    catalog_item_list.append(catalog_item)
+    collection = appliance.collections.catalog_bundles
+    catalog_bundle = collection.create(name='test_{}'.format(fauxfactory.gen_alphanumeric()),
+                                       catalog_items=catalog_item_list,
```

Comment:
catalog_items=[catalog_item.name]
No  need for  catalog_item_list = [] if there is only one catalog item